app/models/Post.scala

```diff
@@ -15,23 +16,27 @@ object Post {
 
 case class Post(slug: String, mainImage: String, content: String, author: Option[Author] = None) {
 
+  val document: Document = Post.asciidoctor.load(content, Options.builder().build())
+
   lazy val htmlContent: String = Post.asciidoctor.convert(
     content,
-    new java.util.HashMap[String, Object]())
-
-  val header: DocumentHeader = Post.asciidoctor.readDocumentHeader(content)
+    Options.builder().build())
 
-  lazy val authorName: String = header.getAuthor.getFullName
+  lazy val authorName: String = document.getAuthors.get(0).getFullName
```

Comment:
There could potentially be more authors (in the future). What do you think about joining them together?

```suggestion
  lazy val authorName: String = document.getAuthors.asScala.map(_.getFullName).mkString(", ")
```

Reply:
Hmm, looking at the rest of the code that would actually break some other things. 

So we might not want to do that (yet)...

Reply:
Introduced authorsNames as suggested by Rui!